probe-dictionary/explorer.css

```diff
@@ -122,3 +122,8 @@ body {
 .hidden {
     display: none;
 }
+
+#detail-cpp-guard:empty:after,#detail-bug-numbers:empty:after{
```

Comment:
Here we use spaces after the `,` and before the `{`.

Reply:
I'll make the changes:) Thanks!